### PR TITLE
Feature/#169 feedback submit visualized

### LINKED ISSUE #169: As a student, I want a hint to be displayed when I already have submitted a feedback text (1).
_Hint: Please review mockup for this story https://www.figma.com/file/zrtdhCTy8UKByCzokQK9XD/Mockup?node-id=0%3A69_

### PR DESCRIPTION
**Pull request by ma-ro**
# Resolves #169 #170  #171 #172

### CODE REVIEW

File: app/controllers/feedbacks_controller.rb [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+    if params[:commit] == "Submit"
+      @feedback = @lecture.feedbacks.create(content: comment_params[:content], user: current_user)
+    elsif params[:commit] == "Update"
+      @feedback = @lecture.feedbacks.where(user_id: current_user.id)
+      @feedback.update(comment_params)
+    end
+    head :no_content
+  end
+
+  def update
```
Comment (m0e33) [inline app/controllers/feedbacks_controller.rb]: Delete update method!

File: app/views/lectures/show.html.erb [hunk 2]
```diff
@@ -1,3 +1,13 @@
+<script>
+
+  function handleSubmit() {
```
Comment (m0e33) [inline app/views/lectures/show.html.erb]: Consider to change location of javascript function.

File: app/views/lectures/show.html.erb [hunk 3]
```diff
@@ -44,19 +54,27 @@
...
-        <p>
-          <%= form.label :content %><br>
-          <%= form.text_field :content %>
-        </p>
-        <p>
-          <%= form.submit %>
-        </p>
+      <%= form_with(model: [ @course, @lecture, @lecture.feedbacks.build ], :html => { :onsubmit => "handleSubmit()" }) do |form| %>
+        <div id="feedback-notification" style="display:none" class="alert alert-success" role="alert"> Feedback submitted</div>
+        <% if @lecture.feedbacks.where(user_id: current_user.id).count >= 1 %>
```
Comment (m0e33) [inline app/views/lectures/show.html.erb]: There is hopefully a better way of checking wether there is already feedback stored for a user.

File: app/views/lectures/show.html.erb [hunk 4]
```diff
@@ -44,19 +54,27 @@
...
+          <p>
+            <%= form.label :content, "You submitted your feedback. Update it here."%>
+            <%= form.text_area(:content, :value => @lecture.feedbacks.where(user_id: @current_user.id).first.content, :class => "form-control") %>
```
Comment (m0e33) [inline app/views/lectures/show.html.erb]: We have to check wether there is a better way of doing this.

File: app/controllers/feedbacks_controller.rb (hunk 1 above)
Comment (ma-ro) [reply app/controllers/feedbacks_controller.rb]: Done

**Pull request merged**